Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -844,15 +857,6 @@ extension TestStore where ScopedState: Equatable, Action: Equatable {
 
 // MARK: - Deprecated after 0.27.1:
 
-extension AlertState.Button {
-  @available(
-    *, deprecated, message: "Cancel buttons must be given an explicit label as their first argument"
-  )
-  public static func cancel(action: AlertState.ButtonAction? = nil) -> Self {
-    .init(action: action, label: TextState("Cancel"), role: .cancel)
-  }
-}
-
```

Comment:
This wasn't compiling so I just removed since it's so old. But I can debug it if we don't want to remove yet.